README.md

```diff
@@ -40,11 +40,11 @@ PARTICULAR PURPOSE. See the [GNU Lesser General Public License] for more details
 
 [crate-image]: https://img.shields.io/crates/v/librpm.svg
 [crate-link]: https://crates.io/crates/librpm
-[build-image]: https://travis-ci.com/RustRPM/librpm.rs.svg?branch=master
-[build-link]: https://travis-ci.com/RustRPM/librpm.rs/
+[build-image]: https://travis-ci.com/rpm-software-management/librpm.rs.svg?branch=master
+[build-link]: https://travis-ci.com/rpm-software-management/librpm.rs/
```

Comment:
Will these work?

Reply:
I'm going to guess no, since it's not doing anything right now...